streamwebs_frontend/streamwebs/views.py

```diff
@@ -244,3 +245,37 @@ def water_quality_edit(request, site_slug):
     return render(request, 'streamwebs/datasheets/water_quality_edit.html', {
         'site': site
     })
+
+
+def macroinvertebrate(request, site_slug, data_id):
+    data = Macroinvertebrates.objects.filter(site_id=site_slug).get(id=data_id)
+    site = Site.objects.get(id=site_slug)
+    return render(request,
+                  'streamwebs/datasheets/macroinvertebrate_view.html', {
+                      'data': data,
+                      'site': site})
+
+
+def macroinvertebrate_edit(request, site_slug):
+    """
+    The view for the submission of a new macroinvertebrate data sheet.
+    """
+    site = Site.objects.get(id=site_slug)
+    added = False
+    if request.method == 'POST':
+        macro_form = MacroinvertebratesForm(data=request.POST)
+
+        if macro_form.is_valid():
+            macro_form = macro_form.save()
+            added = True
+
+        else:
+            print macro_form.errors
```

Comment:
Can you delete this print statement? We've been asked by a couple of people to have cleaner test output if all tests pass